lib/setup.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+
+from distutils.core import setup
+from pathlib import Path
+
+package_name = 'wit'
+
+assets = [
+    "scala-bridge-fetcher_2.12-0.1.0.jar"
+]
+
+scripts = [
+    'bin/wit',
+]
+
+setup(
+    name=package_name,
```

Comment:
Per https://packaging.python.org/guides/distributing-packages-using-setuptools/#name, this is supposed to be the name of the package as listed on PyPI. Given that PyPI already has a [separate package by the name of `wit`](https://pypi.org/project/wit/), I don't think we should name this `wit`, since it could confuse any tools that assume that the value of this field can be used to query PyPI for the package.

Note that the name field here is not the same thing as the name that you use in `import` statements, since hyphens and periods are explicitly allowed, despite not being valid in Python identifiers. Also, for a real-life example of this, see the [PyYAML library](https://github.com/yaml/pyyaml/blob/2f463cf5b0e98a52bc20e348d1e69761bf263b86/setup.py#L2), which has a PyPI name of `PyYAML` but an import package name of `yaml`.

If no one has any better suggestions, I will suggest a package name of `wit-sifive`.